pkg/hoist/hoist_launchable.go

```diff
@@ -322,10 +322,16 @@ func (hl *Launchable) Install(verifier auth.ArtifactVerifier) error {
 	return err
 }
 
-// The version of the artifact is currently derived from the location, using
-// the naming scheme <the-app>_<unique-version-string>.tar.gz
+func (hl *Launchable) tempFilePath() string {
```

Comment:
the `path.Base(hl.location)` call doesn't make sense anymore now that `Location` is a url.URL. I could use the path but in the future the artifact URL path may not be suitable for making a unique temp directory path